Misc/NEWS.d/next/Core_and_Builtins/2023-04-29-23-15-38.gh-issue-103997.BS3uVt.rst

```diff
@@ -0,0 +1,4 @@
+String arguments passed to "-c" are now automatically dedented as if by
+:func:`textwrap.dedent`. This allows "python -c" invocations to be indented
```

Comment:
We're not using `textwrap.dedent` per se but a re-implementation of it in C. If users hack into `textwrap.dedent`, they won't see it being called.

Reply:
This says 'as if by', which to me implies an equivalent algorithm but not using the actual function.

Reply:
I... shouldn't review when I'm tired. I totally missed the "as if" and read "dedented by". Forget this oversight, my bad!